spec/features/user_groups_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+feature "Group" do
+  scenario "user can create a group" do
+    user = create(:user)
+    group_name = "scotty"
+    group_description = "Beam me up"
+
+    sign_in(user)
+    visit dashboard_path
+    click_link "Create Group"
+    fill_in "Name", with: group_name
```

Comment:
This is great. Consider taking a look at https://github.com/thoughtbot/formulaic in the future